src/buildHTML.js

```diff
@@ -1,8 +1,8 @@
 /**
  * This file does the main work of building a domTree structure from a parse
  * tree. The entry point is the `buildHTML` function, which takes a parse tree.
- * Then, the buildExpression, buildGroup, and various groupTypes functions are
- * called, to produce a final HTML tree.
+ * Then, the buildExpression, buildGroup, and various groupBuilders functions
```

Comment:
Remaining `groupTypes` to `groupBuilders` is an improvement.

Reply:
Thoroughly agree! That was @ylemkimon's idea in the last PR.